### DISCUSSION TITLE
[灌水]绳网中冉冉升起的明日之星

### DISCUSSION THREAD

**Discussion by lantainAirblue** [share121/inter-knot#4618] [灌水]
明天的传奇绳匠，以法厄同为目标而奋斗。你们的委托我接下了！我要将岚天的名号打响整个新艾莉都！
![IMG_20240908_155734](https://github.com/user-attachments/assets/c0070bee-aa0a-42ec-aeba-fc4ebb6d197e)

**Comment by lantainAirblue** [comment]
嘤嘤嘤没有人

**Reply by SmogaSmk** [reply]
有，怎么贴照片

**Reply by lantainAirblue** [reply]
这个委托我接下了
![Uploading Screenshot_20240909_001940.jpg…]()
